### DISCUSSION TITLE
Highlight code completion?

### DISCUSSION THREAD

**Discussion by edhemphill** [zed-industries/zed#19221] [Feature Requests]
With the significant amount of configuration in Zed, I was wondering if there is a way to highlight / box / color code completion coming from supermaven or github copilot? 

Sometimes the amount of code provided can be very confusing to look at when it just pops on the screen - and its not super fast to tell what its dropped in and what was below it.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
